.dockerignore

```diff
@@ -1,2 +1,3 @@
 *
 !target/release/maker
+!target/x86_64-unknown-linux-musl/release/maker
```

Comment:
was not on purpose

Reply:
what Docker container we will run it in? with `musl`, everything libc should be statically linked, so we could use a minimal one I guess ;)

Reply:
I just needed this to be able to cross-compile locally :) We are using the standard linux target in our container